SeaPublicWebsite/Views/EnergyEfficiency/ActionPlan/ActionPlanWithDiscardedRecommendations.cshtml

```diff
@@ -25,17 +25,10 @@
         @{ await Html.RenderPartialAsync("ActionPlan/Partials/_AvailableSchemesOrGrants", Model); }
         @{ await Html.RenderPartialAsync("ActionPlan/Partials/_DiscardedRecommendations", Model); }
         
-        <h2 class="govuk-heading-l govuk-!-margin-bottom-3 govuk-!-margin-top-6">
+        <h2 class="govuk-heading-l govuk-!-margin-bottom-3 govuk-!-margin-top-7">
             Other ways to save energy
         </h2>
         
-        <p class="govuk-body">
```

Comment:
Is this meant to be removed?

Reply:
Yes, duplicated information. The one below it is the up to date wording. (On the action plan with saved / maybe there's no duplication, just here)